docs/getting_started/getting_started.md

```diff
@@ -7,30 +7,40 @@
 ## How do I use a pretrained model to predict an image?
 
 ```python
-from deepforest import main
-from deepforest import get_data
-import matplotlib.pyplot as plt
+from deepforest import main, get_data
+from deepforest.visualize import plot_results
+import os
 
 model = main.deepforest()
 model.use_release()
 
 sample_image_path = get_data("OSBS_029.png")
-img = model.predict_image(path=sample_image_path, return_plot=True)
-
-#predict_image returns plot in BlueGreenRed (opencv style), but matplotlib likes RedGreenBlue, switch the channel order. Many functions in deepforest will automatically perform this flip for you and give a warning.
-plt.imshow(img[:,:,::-1])
+results = model.predict_image(path=sample_image_path)
+plot_results(results, root_dir=os.path.dirname(sample_image_path))
```

Comment:
Don't need `root_dir` here since `predict_image` was called with path and should have `root_dir` as an attribute. This also means we don't need the `os` import